### PR TITLE
Add ol8 ospp draft profile

### PR DESCRIPTION
**Pull request by iokomin**
#### Description:
Add OL8 OSPP draft profile implementation

Changes:
- Added OL8 type to supported platforms in OSPP rules (where applicable added OL7 product as well)
- Updated fips and certification rules description to match new product
- Fixed multiline formatting in rules where needed
- In several grub2 rules description where applicable added missing rhel8 to macro conditions

Note: I understand set of changes is huge comparing to majority of review requests however in fact it is only product type extension in existing rules with applicable ol7 and ol8 products. Just let me know if review of this change set should be split up in parts.

#### Testing:
- Checked ol7, ol8 and all build targets to pass
- Checked rules and remediation scripts to work on target OL8 platform
- Checked generated guide content

### CODE REVIEW

**Comment by redhatrises** [comment]
@iokomin the bigger question for this PR is if OL8 going through NIAP and Commons Criteria testing and certification?

**Comment by iokomin** [comment]
@redhatrises while OL7 is FIPS validated and CC certified unfortunately I can't comment on non-public plans of Oracle regarding OL8. Please let me know if there is any restriction or guidance I should follow adding profiles applicable to Oracle Linux.

File: ol8/profiles/ospp.profile [hunk 1]
```diff
@@ -0,0 +1,207 @@
+documentation_complete: true
+
+title: 'OSPP - Protection Profile for General Purpose Operating Systems'
```
Comment (redhatrises) [inline ol8/profiles/ospp.profile]: @iokomin I would change the title to `[DRAFT] OSPP - Protection Profile for General Purpose Operating Systems`, and then when OL is OSPP approved, remove `[DRAFT]`

File: ol8/profiles/ospp.profile [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+description: |-
+    This profile reflects mandatory configuration controls identified in the
+    NIAP Configuration Annex to the Protection Profile for General Purpose
+    Operating Systems (Protection Profile Version 4.2).
```
Comment (redhatrises) [inline ol8/profiles/ospp.profile]: @iokomin I would add the following to the description:

```
This profile is currently under review. Use of this profile does not denote or guarantee NIAP approval or certification
until this profile has been approved by NIAP.
```
Then when OL is NIAP validated and certited, remove the above sentences.

Comment (iokomin) [reply ol8/profiles/ospp.profile]: @redhatrises please find draft note added to profile description.

File: ol8/profiles/ospp.profile (hunk 1 above)
Comment (iokomin) [reply ol8/profiles/ospp.profile]: @redhatrises please find [DRAFT] added to profile title.

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by iokomin** [comment]
@jan-cerny please find updated branch, sanity build 'make all' passed

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by jan-cerny** [comment]
@iokomin Thanks!

**Comment by shawndwells** [comment]
@openscap-ci add to whitelist

**Pull request merged**